lib/justice_gov_sk/infrastructure.rb

```diff
@@ -0,0 +1,19 @@
+module JusticeGovSk
+  module Infrastructure
+    # TODO implement async?
+
+    def crawl
+      (1..pages).each do |page|
+        url = uri.build(page: page)
```

Comment:
dobre, asi to nechajme takto... ja som to pomenoval uri vtedy preto, lebo som sa tvaril, ze zdrojom dat nemusi byt nutne nieco na webe